CONTRIBUTING.md

```diff
@@ -57,27 +81,38 @@ During markdown to HTML conversion, some predefined variables located in the [va
 For instance, the `varVersion` variable matches the current documentation version.
 To reference this variable, use the `${varVersion}` notation in _.md_ files.
 
+#### Markdown titles
+
+In markdown, lines starting with `#` are converted to `<hx>` tags where _x_ is the number of `#` : 
+* use `#` for `h1` tag
+* use `##` for `h2` tag
+* use `###` for `h3` tag
+* etc... up to `h6`
+
+Checkout [the markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes) for more information.
+
 ### Table of content
 
-A table of content (ToC) is automatically added on each page. It will be placed right before the first `h2` title.
+A table of content (ToC) is automatically generated on each page from `##` and `###` titles (converted as `h2` and `h3` html tags).  
 
-Automatically generated ToC includes links to `h2` and `h3` titles.
+It will be placed right before the first `h2` title.  
 
 ### Quick note on "howtos"
 
 When you create **howtos** make sure that:
   - In the taxonomy, their titles start with verbs, so it finishes the sentence "how to...", like "Manage multiple references in Business Objects".
   - In the file itself, the title includes the "how to" phrase, like "How to manage multiple references in Business Objects".
   - You reference the how to page in other descriptive pages of the existing taxonomy, to optimize its discoverability.
+  - the Howtos are a step by step guide to manage a use case. This use case should be described briefly as an introduction (following the page title but without its own title) for the user to understand the goal of the howto.
 
-## How to add images?
+## Markdown syntax
 
-All images must be added to the **md/images** folder.
+Current conversion to html is done with `markdown-it` with the [Github-flavored Markdown](https://help.github.com/categories/writing-on-github/). Checkout [the cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes) for more information.
```

Comment:
Use this link instead ? https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet